cluster/gce/container-linux/configure-helper.sh

```diff
@@ -799,6 +800,21 @@ function start-kube-apiserver {
     params+=" --etcd-quorum-read=${ETCD_QUORUM_READ}"
   fi
 
+  if [[ "${ENABLE_APISERVER_BASIC_AUDIT:-}" == "true" ]]; then
+    # We currently only support enabling with a fixed path and with built-in log
+    # rotation "disabled" (large value) so it behaves like kube-apiserver.log.
+    # External log rotation should be set up the same as for kube-apiserver.log.
+    params+=" --audit-log-path=/var/log/kube-apiserver-audit.log"
+    params+=" --audit-log-maxage=0"
+    params+=" --audit-log-maxbackup=0"
+    # Lumberjack doesn't offer any way to disable size-based rotation. It also
+    # has an in-memory counter that doesn't notice if you truncate the file.
+    # 1000000000 (MiB) is a large number that fits in 31 bits. If the log grows
+    # by 1MiB/s, it will rotate after ~31 years if apiserver never restarts.
```

Comment:
The idea here is to disable the audit log feature's built-in log rotation (Lumberjack) so we can use the same `logrotate` facility we already have for `kube-apiserver.log`.

The current `logrotate` settings rotate everything matching the wildcard `/var/log/*.log` so this new `/var/log/kube-apiserver-audit.log` is included. I verified that it rotates as expected.

https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh#L124